include/Controller.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include "Board.h"
+#include "Chess.h"
+
+class Controller {
+public:
+	Controller();
+	void run();
+private:
+	Chess m_chess;
+	Board m_board;
+	string m_stringBoard;

```

Comment:
You should initialize the m_stringBoard member variable in the Controller constructor instead of declaring it as a separate variable. This way, you ensure that the string representation of the board is always in sync with the Board object.